hometask_18.py

```diff
@@ -0,0 +1,43 @@
+class Bot:
+    def init(self, name):
+        self.name = name
+
+    def say_name (self):
+        print(self.name)
+
+    def send_message (self, message):
+        print(message)
+
+class TelegramBot(Bot):
+
```

Comment:
add a constructor here and add chat_id and url as class variables, or you can initialize these variables in the constructor (None by default)